packages/table/test/tableTests.tsx

```diff
@@ -329,52 +329,64 @@ describe("<Table>", () => {
                 ReactDOM.unmountComponentAtNode(container);
             });
 
-            it("syncs quadrant scroll offsets when scrolling the main quadrant", () => {
+            it("syncs quadrant scroll offsets when scrolling the main quadrant", (done) => {
                 // simulating a "scroll" or "wheel" event doesn't seem to affect the
                 // scrollTop/scrollLeft the way it would in practice, so we need to tweak those
                 // explicitly before triggering.
                 mainScrollContainer.scrollLeft = SCROLL_OFFSET_X;
                 mainScrollContainer.scrollTop = SCROLL_OFFSET_Y;
                 TestUtils.Simulate.scroll(mainScrollContainer);
 
-                assertScrollPositionEquals(topScrollContainer, SCROLL_OFFSET_X, 0);
-                assertScrollPositionEquals(leftScrollContainer, 0, SCROLL_OFFSET_Y);
-                assertScrollPositionEquals(topLeftScrollContainer, 0, 0);
+                requestAnimationFrame(() => {
```

Comment:
Had to get a little cute here putting the tests themselves in `rAF`, to give the scroll logic a chance to execute.